1-SendingAndStandardOut/README_RU.md

```diff
@@ -0,0 +1,110 @@
+# Отправка сообщений и страндартный Out
+
+## Скажем Привет
+
+!["Person waiving hello"](helloWorld.png)
```

Comment:
You may also translate the image captions if you like.

Reply:
Yeah, I will